app/controllers/users_controller.rb

```diff
@@ -1,10 +1,8 @@
 class UsersController < ApplicationController
 
-  skip_before_filter :registration,
-                     only: [:ask_for_email, :edit, :update]
+  skip_before_filter :registration, only: [:ask_for_email]
```

Comment:
Why did `:edit` and `:update` go away here?  I think they are needed because the edit screen is now open _during_ registration (whereas it didn't used to be).